### PR TITLE
LPD-43090 Create tests for Categories

### PR DESCRIPTION
**Pull request by DiegoHu97**
LPD-43090 Create tests for Categories
# What is this trying to solve?
Create the following test cases:
- AddCategories
- EditCategory
- DeleteCategoryByMenu
- ViewDescriptionOfVocabulary
- MoveBetweenVocabularies

[Link to ticket](https://liferay.atlassian.net/browse/LPD-43090)

# How can I test it?

Run the tests

### CODE REVIEW

File: modules/test/playwright/tests/asset-categories-admin-web/pages/AssetCategoriesEditPage.ts [hunk 1]
```diff
@@ -60,12 +57,42 @@ export class AssetCategoriesEditPage {
...
+			await this.save();
+		}
+	}
+
+	async fillName(name: string) {
+		await this.descriptionField.waitFor();
+		await this.nameInput.fill(name);
+	}
+
+	async goto(action: string, title: string) {
```
Comment (boton) [inline modules/test/playwright/tests/asset-categories-admin-web/pages/AssetCategoriesEditPage.ts:70]: This goto may need to be renamed to goToAction and moved to AssetCategoriesAdminPage.ts

File: modules/test/playwright/tests/asset-categories-admin-web/pages/VocabulariesEditPage.ts [hunk 2]
```diff
@@ -42,6 +49,10 @@ export class VocabulariesEditPage {
 		await this.deleteButton.click();
 	}
 
+	async fillDescription(description: string) {
+		await this.descriptionInput.fill(description);
+	}
+
```
Comment (boton) [inline modules/test/playwright/tests/asset-categories-admin-web/pages/VocabulariesEditPage.ts:55]: Not needed as a method, I think we can inline it

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by DiegoHu97** [comment]
ci:test:content-management-playwright

**Review by boton — APPROVED** [review]
LGTM

**Comment by DiegoHu97** [comment]
ci:forward

**Pull request closed without merging**